src/common/opencl.c

```diff
@@ -3485,7 +3485,7 @@ void dt_opencl_check_tuning(const int devid)
   else
   {
     // calculate data from fractions
-    const size_t disposable = allmem - 400ul * 1024ul * 1024ul;
+    const size_t disposable = allmem - 600ul * 1024ul * 1024ul;
```

Comment:
For better readability and discoverability of this feature I'd propose to have a defined (in opencl.h) named RESERVED_HEADROOM (or something like that) set to 600 and used it here. What do you think?

Reply:
:-) We already have DT_OPENCL_DEFAULT_HEADROOM for that purpose